pkg/lib/host-connector/util/aik_quote_verifer.go

```diff
@@ -340,7 +343,7 @@ func createPCRManifest(pcrList []string, eventLog string) (types.PcrManifest, er
 			}
 		}
 	}
-	pcrManifest.PcrEventLogMap, err = getPcrEventLog(eventLog)
+	pcrManifest.PcrEventLogMapNew, err = getPcrEventLog(eventLog)
```

Comment:
any requirement to change this name?

Reply:
For linux flow, using the new structure as pcr_eventlog_map structure is changed.

Reply:
PcrEventLogMapNew does not describe if it's linux flow